triggers/runner/src/main/scala/com/digitalasset/daml/lf/engine/trigger/Runner.scala

```diff
@@ -671,30 +713,32 @@ private[lf] class Runner private (
       }
 
       UnfoldState(v)({ state =>
-        numberOfRuleEvaluations += 1
```

Comment:
Bug fix to stop double counting - this increment is already performed in the `go` function